charts/sonarqube/templates/init-fs.yaml

```diff
@@ -12,6 +12,9 @@ data:
     {{- if .Values.plugins.install }}
     chown -R {{ .Values.persistence.uid }}:{{ .Values.persistence.guid }} {{ .Values.sonarqubeFolder }}/extensions/plugins
     {{- end }}
+    {{- if .Values.jdbcOverwrite.oracleJdbcDriverURL }}
+    chown -R {{ .Values.persistence.uid }}:{{ .Values.persistence.guid }} {{ .Values.sonarqubeFolder }}/extensions/jdbc-driver/oracle
```

Comment:
not tested yet, but does this work knowing that this path might not have been created yet? ha maybe yes because on the volume path definition it will automatically create the subpath ? will test

Reply:
I tested it and it doesn't seem to be a problem. :)